censys/common/utils.py

```diff
@@ -17,3 +17,17 @@ def format_rfc3339(time: Datetime) -> str:
     if isinstance(time, (datetime.date, datetime.datetime)):
         time = time.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
     return time
+
+
+def format_iso8601(time: Datetime) -> str:
```

Comment:
surprised this isnt part of the regular ol datetime library somewhere

Reply:
Yeah, we don't use the full ISO 8601, otherwise we could use `date.isoformat()`